src/OrdersList/OrdersListFilterConfig.js

```diff
@@ -0,0 +1,37 @@
+import { FILTERS } from './constants';
+
+const filterConfig = [
+  {
+    name: FILTERS.ASSIGNED_TO,
+    cql: FILTERS.ASSIGNED_TO,
+    values: [],
+  },
+  {
+    name: FILTERS.STATUS,
+    cql: FILTERS.STATUS,
+    values: [],
+  },
+  {
+    name: FILTERS.RECEIPT_STATUS,
+    cql: FILTERS.RECEIPT_STATUS,
+    values: [],
+  },
+  {
+    name: FILTERS.DATE_ORDERED,
+    cql: FILTERS.DATE_ORDERED,
+    isRange: true,
+    isEqual: true,
+    rangeSeparator: ':',
+    values: [],
+  },
+];
+
+const SearchableIndexes = [
+  { label: 'All', value: 'all', makeQuery: term => `(id="${term}*" or poNumber="${term}*" or vendor="${term}*" or assignedTo="${term}*")` },
```

Comment:
Are these labels translated?

Reply:
`SearchableIndexes` is not used (I don't know why)

forgot to remove